roles/ckan/tasks/azure-deploy.yml

```diff
@@ -0,0 +1,103 @@
+---
+
+- name: Get subscription_id for later
+  azure.azcollection.azure_rm_subscription_info:
+    all: true
+  register: subscription_output
+
+- name: Create CKAN namespace
+  command: >
+    az aks command invoke --resource-group {{ resource_group_name }} --name {{resource_prefix}}-aks \
+    --command "kubectl create namespace {{ application_namespace }}"
+
+- set_fact: 
+    storage_class_name: managed-csi # Azure
+
+- name: Get datalake info
+  azure.azcollection.azure_rm_storageaccount_info:
+    resource_group: "{{ resource_group_name }}"
+    name: "{{ storage_account_name }}"
+  register: datalake_info_output_raw
+
+- name: Log datalake info
+  debug: "msg={{ datalake_info_output_raw }}"
+
+- name: Get private link service info by name
+  azure.azcollection.azure_rm_privatelinkservice_info:
+    resource_group: "{{ resource_group_name }}"
+    name: "{{resource_prefix}}-pe-storage"
+  register: datalake_pl_output_raw
+
+- name: Log datalake info
+  debug: "msg={{ datalake_pl_output_raw }}"
+
+- name: Show database info
+  command: >
```

Comment:
Using `azure.azcollection.azure_rm_postgresqlserver_info` instead of a `command` here would be recommended.

Reply:
`azure.azcollection.azure_rm_postgresqlserver_info` is for single instance postgres instances, an outdated API. 

However, https://github.com/ansible-collections/azure/issues/714 seems to have now been shipped so can swap out all the DB commands for `azure_rm_postgresqlflexibledatabase_` etc :+1:

Reply:
https://fjelltopp.atlassian.net/browse/WAC-96?atlOrigin=eyJpIjoiNTkxMzEwNzQyYmUzNDM3NmJhYTNhM2FiZWFiMTFiYjIiLCJwIjoiaiJ9